index.d.ts

```diff
@@ -0,0 +1,18 @@
+import type { JSONSchema7 } from 'json-schema';
+
+declare module '@asyncapi/specs' {
```

Comment:
Are we gonna remove `index.js` in favor of this?

Reply:
but then we need transpilation TS -> JS process.

Reply:
Yes, otherwise we manually maintain both versions.

Reply:
Can be done, lets wait for others opinion.

Reply:
No longer current. PR has changed its "character".

Reply:
yeah, I definitely prefer we generate `index.js` as it doesn't make sense to manually duplicate

Reply:
Are we gonna do this in this PR or in another one right after merging this one?

Reply:
Due to the fact this PR blocks https://github.com/asyncapi/spec-json-schemas/pull/264 (See [here](https://github.com/asyncapi/spec-json-schemas/pull/264#issuecomment-1273027917)), I suggest we first merge this one, we then enable TS transpilation asap in another PR